testsettings.py

```diff
@@ -122,3 +122,5 @@ def _set_logging_levels(levels):
 ES_SEARCH_TIMEOUT = 5
 
 FORMPLAYER_INTERNAL_AUTH_KEY = "abc123"
+
+helper.update_redis_location_for_tests(CACHES)  # noqa: F405
```

Comment:
nit: What do you think of moving this up beside `helper.assign_test_db_names(DATABASES)` since they are doing similar things?